src/plasma_manager.c

```diff
@@ -103,6 +79,9 @@ struct client_object_connection {
   char **manager_vector;
   /** The number of manager locations in the array manager_vector. */
   int manager_count;
+  /** The index of the next manager in manager_vector that we should try to
+   * contact. */
+  int next_manager;
```

Comment:
Yes, there should always be a next manager to try, so I didn't think that needed documentation, but I can make a note that it's dependent on the retry policy.